dotcom-rendering/src/components/MostViewedRight.tsx

```diff
@@ -29,47 +30,91 @@ interface Props {
 	stickToTop?: boolean;
 }
 
+interface MostViewedRightPayloadType {
+	tabs: FETrailTabType[];
+}
+
 export const MostViewedRight = ({
 	limitItems = 5,
 	stickToTop = false,
 }: Props) => {
+	const ABTestAPI = useAB()?.api;
+	const inDeeplyReadTestVariant =
+		ABTestAPI?.isUserInVariant('DeeplyReadTest', 'deeply-read-only') ??
+		false;
+	const inMostViewedDeeplyReadTestVariant =
+		ABTestAPI?.isUserInVariant(
+			'DeeplyReadTest',
+			'deeply-read-and-most-read',
+		) ?? false;
```

Comment:
It might be clearer and more concise to coerce the values to boolean with the [double NOT operator `!!`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_NOT)

As it's only possible to be in a single variant, it would probably be clearer to model this as a union type, where the state is either `most-read-only`, `deeply-read-only` or `deeply-read-and-most-read`.